### DISCUSSION TITLE
v1.6.8.1-beta

### DISCUSSION THREAD

**Discussion by deniscerri** [deniscerri/ytdlnis#300] [General]
## What's new

- include search history when searching
- removed scroll bug from command tab
- added spacing between command template title and ok button in selection card
- made download progress not dublicate in terminal
- made ability to store terminal state
- added ability to create multiple terminal instances and show them as a list similar to download queue
- fixed thumbnail download not working
- fixed app crashing when clicking on format updated notification
- fixed app crashing when double clicking format on multiple download card
- added custom sponsorblock api preference
- removed contextual app bar when you its enabled and the user taps the log button in the erorred tab
- made app always show quick download card and asynchoronously load data. Quick Download now if its on, it wont load data at all
- added shimmer when loading data in the download card
- fixed app showing no formats if there were no common formats. Now it will give you generic formats
- made open command template list be half the screen, shortcuts third of the screen so the user can see what its being added
- fixed sometimes app slipping queued downloads even though its told to pause all
- fixed trim filenames cutting files too short
- made mediastore scanning of files one by one
- fixed filename template not working in multiple download card
- fixed -F in terminal not being inline
- added preferred audio codec
- made auto update on boot if there are no active downloads
- fixed format text overlapping
- added a new error activity dialog in cases yt-dlp data fetching in the home screen fails. You can copy the log

## Autogenerated changed
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/283
* Create README-id.MD (Indonesian README) by @teddysulaimanGL in https://github.com/deniscerri/ytdlnis/pull/291
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/288
* Update README-pt.md by @gigoloinc in https://github.com/deniscerri/ytdlnis/pull/293
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/292

## New Contributors
* @teddysulaimanGL made their first contribution in https://github.com/deniscerri/ytdlnis/pull/291
* @gigoloinc made their first contribution in https://github.com/deniscerri/ytdlnis/pull/293

**Full Changelog**: https://github.com/deniscerri/ytdlnis/compare/v1.6.7...v1.6.8.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/deniscerri/ytdlnis/releases/tag/v1.6.8.1-beta'>v1.6.8.1-beta</a>.</em>

**Comment by neonnnnn-o** [comment]
Excuse me, I log in the youtube account, but when I want to download the members only video, it still let me join the channel.(But I already have that channal's member.) How should I do😭😭

**Reply by deniscerri** [reply]
I don't get it. Can you explain in more detail what you are doing

**Comment by neonnnnn-o** [comment]
I wanna download the members only video, so I have to use the cookies right?

I was login and get the cookies, after I press the download button
[picture1], this will come out [picture 2].

Yesterday I even can see the [picture 1], when I search the video, it just
tell me "join the channel to get this video".

It happens on download video or audio.

So I don't know which step is wrong😭

(I'm not the English speaker, I try my best, sorry to bother you😭😭)

Thanks for help!!

deniscerri ***@***.***> 於 2023年10月30日 週一 01:20 寫道：

**Reply by deniscerri** [reply]
there are no pictures friend

**Comment by neonnnnn-o** [comment]
omg sorry I reply you from email😭
here's the picture

**Reply by deniscerri** [reply]
Judging from the error it looks like the app has problems accessing that folder. Looks like its an SD CARD.
What you can do
1. Turn on Allow access to all directories in the folders settings
2. Turn on "Cache downloads first"
3. Try to change the download path after these 2 steps and try again

**Reply by neonnnnn-o** [reply]
OMG It works!!!!
You've saved my life, I can't thank you enough🥹🥹
